domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandlingsinformasjon.kt

```diff
@@ -0,0 +1,337 @@
+package no.nav.su.se.bakover.domain.behandling
+
+import no.nav.su.se.bakover.domain.Boforhold
+import no.nav.su.se.bakover.domain.dto.DtoConvertable
+
+data class Behandlingsinformasjon(
+    private var uførhet: Uførhet? = null,
```

Comment:
Kunne disse egentlig vært public val? Også slette Dtoene?

Reply:
Det hadde vært veldig deilig, men jeg tror jeg da fikk med properties for funksjonene (`avslag`, `complete`, `innvilget`). Hva kan jeg gjøre for å unngå det?

Reply:
Så Jackson tar med de funksjonene når den serialiserer til databasen? Jeg ville prøvd noen annotasjoner som @JsonIgnore

Reply:
Hm, det kan se ut som at jeg enten husker feil eller har gjort en endring siden jeg så de feltene, for nå virker det som at det funker fint med å droppe alle DTO-ene. Kommer commit straks.